common-libs/internal-api-clients/src/services/AffinidiVaultApiService.ts

```diff
@@ -24,10 +24,15 @@ export default class AffinidiVaultApiService extends GenericApiService<ApiSpec>
   }
 
   async searchCredentials(accessToken: string, storageRegion: string, types: string[][]) {
+    const queryParams: any = {}
```

Comment:
Please avoid using `any` whenever possible, because we lose type checking that way.

One of the alternative approaches:
```
queryParams: (types.length > 0) ? { types: JSON.stringify(types) } : {},
```